docs/content/en/docs/clustermgmt/observability/expose-metrics.md

```diff
@@ -0,0 +1,146 @@
+
+To configure a proxy for exposing metrics on an EKS-A cluster, you can perform the following steps:
+
+1.  Create a new cluster role for the client to access the metrics endpoint of the components.
+    ```bash
+    cat << EOF | kubectl apply -f -
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: metrics-reader
+    rules:
+      - nonResourceURLs:
+          - "/metrics"
+        verbs:
+          - get
+    EOF
+    ```
+
+2.  Create a new cluster role binding to bind the above cluster role to the client pod's service account.
+    
+    Below is an example cluster role binding if you use prometheus as the client to scrape metrics.
+    ```bash
+    cat << EOF | kubectl apply -f -
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: metrics-reader-binding
+    subjects:
+    - kind: ServiceAccount
+      name: prometheus-server
+      namespace: observability
+    roleRef:
+      kind: ClusterRole
+      name: metrics-reader
+      apiGroup: rbac.authorization.k8s.io
+    EOF
+    ```
+
+3.  Create a config map to store the proxy configuration.
+    
+    Below is an example config map if you use HAProxy as the proxy server.
+    ```bash
+    cat << EOF | kubectl apply -f -
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: metrics-proxy
+    data:
+      haproxy.cfg: |
+        defaults
+          mode http
+          timeout connect 5000ms
+          timeout client 5000ms
+          timeout server 5000ms
+          default-server maxconn 10
+
+        frontend kube-proxy
+          bind ${NODE_IP}:10249
+          http-request deny if !{ path /metrics }
+          default_backend kube-proxy
+        backend kube-proxy
+          server kube-proxy 127.0.0.1:10249 check
+
+        frontend kube-controller-manager
+          bind ${NODE_IP}:10257
+          http-request deny if !{ path /metrics }
+          default_backend kube-controller-manager
+        backend kube-controller-manager
+          server kube-controller-manager 127.0.0.1:10257 ssl verify none check
+
+        frontend kube-scheduler
+          bind ${NODE_IP}:10259
+          http-request deny if !{ path /metrics }
+          default_backend kube-scheduler
+        backend kube-scheduler
+          server kube-scheduler 127.0.0.1:10259 ssl verify none check
+    EOF
+    ```
+
+4.  Create a daemonset for the proxy and mount the config map volume onto the proxy pods.
+    
+    Below is an example configuration for the HAProxy daemonset.
```

Comment:
I would omit this sentence. Or just add all of this under a subsection (##HAProxy)